charts/microgateway/tests/__snapshot__/configmap_test.yaml.snap

```diff
@@ -0,0 +1,36 @@
+expert mode:
```

Comment:
**subjective opinion**
I prefer when file name start with test_

Reply:
I stuck to the naming conventions promoted by the unit testing tool we use. In this way, the tool is able to find the tests to run automatically. If we change the pattern, we would have to provide the test spec files to run in every invocation.